### ISSUE TITLE
Disabled attribute on DropButton: Field not greyed-out

### ISSUE THREAD

**Issue by LiveFreeAndRoam** [grommet/grommet-theme-hpe#91]
This shows the disable attribute on the DropButton is not causing the button to be greyed out.

## Using grommet-theme-hpe@^1.0.5:
The dropdown buttons are correctly showing greyed-out, since they are disabled.

![image](https://user-images.githubusercontent.com/8231064/89841574-a7903c80-dbb6-11ea-85c7-c74381f47b8f.png)

## Using the NEXT-stable branch... 

```
    "grommet-theme-hpe": "https://github.com/grommet/grommet-theme-hpe/tarball/NEXT-stable",
```

Dropdown is disabled (can't be clicked) but does not show as greyed out.

![image](https://user-images.githubusercontent.com/8231064/89841652-dd352580-dbb6-11ea-84b1-fe17b2c33d26.png)

**Comment by ShimiSun** [comment]
linking to https://github.com/hpe-design/design-system/issues/1051

**Comment by taysea** [comment]
What I'm noticing here is that Select icon takes a color, which we have defined as `text`. With buttons, we are defined their disabled behavior to have color of `text-weak`: https://github.com/grommet/grommet-theme-hpe/blob/NEXT/src/js/index.js#L327, however, this color is not being applied to the icon in Select. Instead, it is being controlled by this line: https://github.com/grommet/grommet/blob/master/src/js/components/Select/Select.js#L188

What seems like the cleanest approach is that the user would be able to set `select.disabled.icons.color` in the theme. This would need to be added in grommet.

**Comment by taysea** [comment]
For this case, it seems like another case that the Select is not receiving disabled styling. Can you try the approach I wrote [on this issue](https://github.com/grommet/grommet-theme-hpe/issues/96#issuecomment-680998493) and confirm it solves this issue? We will add it to the theme if so.

**Comment by ShimiSun** [comment]
The issue is fixed on v2.0.0. feel free to reopen or file a new issue if needed.
